api/models/vpp/interfaces/state.proto

```diff
@@ -1,8 +1,13 @@
 syntax = "proto3";
 
-package interfaces;
+package vpp.interfaces;
 
-import "interface.proto";
+option go_package = "github.com/ligato/vpp-agent/api/models/vpp/interfaces;vpp_interfaces";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+import "vpp/interfaces/interface.proto";
 
 message InterfaceState {
```

Comment:
Curious, why a string and not an enum?

Reply:
Not sure what you are referring to here. This `InterfaceState` is remnant from the old behavior and might be deprecated to provide uniform way for the state of any configuration.

Reply:
"History" is always a reason :)  I was mostly curious as to the reason, not arguing one way or the other.  I've seen State handled both ways :)